plugin/src/test/java/io/jenkins/plugins/casc/CascJmhBenchmarkStateTest.java

```diff
@@ -0,0 +1,40 @@
+package jenkins.benchmark.jmh;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.TimeUnit;
+import org.junit.Test;
+import org.openjdk.jmh.results.format.ResultFormatType;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.ChainedOptionsBuilder;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+/**
+ * Runs sample benchmarks from JUnit tests.
+ */
+public class CascJmhBenchmarkStateTest {
+    @Test
+    public void testJmhBenchmarks() throws Exception {
```

Comment:
this test is missing an assert :sweat:

Reply:
@casz Actually, this test does call asserts. Please see: https://github.com/jenkinsci/configuration-as-code-plugin/blob/28888a2754c723b710c978fae3433ae1c0aa7e29/plugin/src/test/java/io/jenkins/plugins/casc/SampleBenchmark.java#L25-L26
Codacy doesn't seem to be smart enough to see it.